src/parse-templates.ts

```diff
@@ -84,6 +90,36 @@ function isEscaped(template: string, _offset: number | undefined) {
   return count % 2 === 1;
 }
 
+export const DEFAULT_PARSE_OPTIONS = {
```

Comment:
Should we also update `parseTemplates` to actually default these options (if nothing was provided)?

@dfreeman / @chriskrycho - What do y'all think?

Reply:
Should we also rename to `DEFAULT_PARSE_TEMPLATES_OPTIONS`? (In case we _someday_ provide both a `parse` and `parseTemplates`...)

Reply:
updated good call.

Reply:
> Should we also update parseTemplates to actually default these options (if nothing was provided)?

Defaulting at the top level (i.e. `options = DEFAULT_PARSE_OPTIONS`) seems reasonable to me 👍